pkg/ebpf/c/common/logging.h

```diff
@@ -1,12 +1,11 @@
-#ifndef __TRACEE_LOGGING_H__
-#define __TRACEE_LOGGING_H__
+#ifndef __COMMON_LOGGING_H__
+#define __COMMON_LOGGING_H__
 
 #include <vmlinux.h>
+#include <vmlinux_flavors.h>
+#include <vmlinux_missing.h>
```

Comment:
Are these headers needed here?

Reply:
ditto (tried to create a pattern, can remove it).